docs/docs/api.md

```diff
@@ -0,0 +1,324 @@
+---
+outline: [2, 3]
+---
+
+# PGlite API
+
+## Main Constructor
+
+`new PGlite(dataDir: string, options: PGliteOptions)`<br/>
+`new PGlite(options: PGliteOptions)`
+
+A new PGlite instance is created using the `new PGlite()` constructor.
```

Comment:
might be worth deciding on whether we want `create` as the main initialization before release?